src/main/java/by/custom/utilcalculator/Main.java

```diff
@@ -9,12 +10,22 @@
 
 public class Main {
     public static void main(String[] args) {
+        Bot localBot = Bot.getBot();
+        singletonInits();
         try {
             //this is the bot's registration - i don't know what is going here, i have just copied this from manual
-            TelegramBotsApi bot = new TelegramBotsApi(DefaultBotSession.class);
-            bot.registerBot(Bot.getBot());
+            TelegramBotsApi telegramBot = new TelegramBotsApi(DefaultBotSession.class);
+            telegramBot.registerBot(localBot);
         } catch (TelegramApiException e) {
             e.printStackTrace();
         }
     }
+
+    private static void singletonInits() {
```

Comment:
I'm not sure that it makes sense.
We will init classes in Main but in a different way.

Reply:
Removed it. Waiting for instructions to fix it. 
To be honest, when I did this, I clearly understood that I was doing useless thing. But I had to do something to try init Singletons in the main class, but I hadn't another ideas how to handle this request except this silly way.